tests/test_dispatcher/test_fsm_context.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from aiogram.contrib.fsm_storage.memory import MemoryStorage
+from aiogram.dispatcher import FSMContext
+
+
+class TestFSMContext:
+    @pytest.mark.asyncio
+    async def test_update_data(self):
+        context = FSMContext(MemoryStorage(), chat=1, user=1)
+        async with context.proxy() as data:
+            data.update(key1="value1", key2="value2")
```

Comment:
It would be nice to get context data again and assert that it's become updated

Reply:
Well... I'm not sure that it is required, since it is a test for MemoryStorage(), but if you'd like to...